monai/apps/reconstruction/transforms/array.py

```diff
@@ -0,0 +1,287 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from abc import ABC, abstractmethod
+from typing import Sequence
+
+import numpy as np
+from torch import Tensor
+
+from monai.apps.reconstruction.complex_utils import complex_abs, convert_to_tensor_complex
+from monai.apps.reconstruction.mri_utils import root_sum_of_squares
+from monai.config.type_definitions import NdarrayOrTensor
+from monai.data.fft_utils import ifftn_centered
+from monai.transforms.transform import Randomizable, Transform
+from monai.utils.enums import TransformBackends
+from monai.utils.type_conversion import convert_to_tensor
+
+
+class KspaceMask(Randomizable, ABC):
```

Comment:
This can inherit from `RandomizableTransform`, which means that it will inherit from `Transform`, `Randomizable` and `ABC`. You can then delete `ABC` here and `,Transform` inheritance from child classes.